src/main/java/frc/utils/auto/AutonomousChooser.java

```diff
@@ -0,0 +1,23 @@
+package frc.utils.auto;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import edu.wpi.first.wpilibj2.command.Command;
+import org.littletonrobotics.junction.networktables.LoggedDashboardChooser;
+
+public class AutonomousChooser {
+
+    private final LoggedDashboardChooser<Command> chooser;
+
+    public AutonomousChooser(String name, String defaultOption) {
+        this.chooser = new LoggedDashboardChooser<Command>(name, AutoBuilder.buildAutoChooser(defaultOption));
+    }
+
+    public AutonomousChooser(String name) {
+        this(name, "");
```

Comment:
if the default option is "" will it crash?

Reply:
No it will just make the default "None"

Reply:
i think its more informative to write "None" instead of "", and even maybe insert it as const or as param

Reply:
If there's an auto that doesn't score notes and you call it "None" it will be chosen, without an option of not doing anything in auto. Empty String is the only name that an auto can't have.

Reply:
I get it, but maybe we will want to put this year only shoot as default if we forget to change auto or something. So i prefer to have the option to change default without changing the class

Reply:
that's what the second constructor is for

Reply:
didn't saw to other one my fault